src/manage.c

```diff
@@ -370,7 +367,7 @@ static u8 manage_track_new_acq(void)
   return MANAGE_NO_CHANNELS_FREE;
 }
 
-/** Clear unhealthy flags after some time.  Flags are reset one per day. */
+/** Clear unhealthy flags after some time.  Flags are reset once per day. */
```

Comment:
This comment is ambigous.  Should say reset by the airforce or government.

I read this as the frequency that the firmware calls this function (which is every 5 times through the code, not oncer per day)